### PR TITLE
Minor improvements over consistency

### PR DESCRIPTION
**Pull request by reyang**
* Make package description consistent
  * I've removed the ending period from description per guidance from the official nuget document.
  * I've made `OpenTelemetry .NET` instead of `OpenTelemetry` as NuGet allows C++ components.

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.Dependencies/OpenTelemetry.Instrumentation.Dependencies.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net452;net461</TargetFrameworks>
-    <Description>OpenTelemetry instrumentation for Http, Sql, &amp; Azure dependencies.</Description>
```
Comment (reyang) [inline src/OpenTelemetry.Instrumentation.Dependencies/OpenTelemetry.Instrumentation.Dependencies.csproj:4]: I've removed `Azure` as it doesn't make sense anymore.

File: src/OpenTelemetry.Extensions.Hosting/OpenTelemetry.Extensions.Hosting.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <Description>OpenTelemetry hosting integration adds startup extensions to register OpenTelemetry into .NET Core applications.</Description>
+    <Description>Startup extensions to register OpenTelemetry into .NET Core applications</Description>
```
Comment (cijothomas) [inline src/OpenTelemetry.Extensions.Hosting/OpenTelemetry.Extensions.Hosting.csproj:4]: This is not tied exclusively to .NET core applications.. Any application using the Msft.Extensions.DependencyInjection and Ms.Extensions.Hosting can leverage this..

Comment (reyang) [reply src/OpenTelemetry.Extensions.Hosting/OpenTelemetry.Extensions.Hosting.csproj:4]: Do you have a suggested wording?

Comment (cijothomas) [reply src/OpenTelemetry.Extensions.Hosting/OpenTelemetry.Extensions.Hosting.csproj:4]: Startup extensions to register OpenTelemetry into .NET applications using Microsoft.Extensions.DependencyInjection and Microsoft.Extensions.Hosting - ?

**Pull request merged**